DQM/L1TMonitorClient/python/L1TStage2BMTFClient_cff.py

```diff
@@ -18,12 +18,12 @@
     binomialErr = cms.untracked.bool(True)
 )
 
-l1tStage2BmtfZeroSuppFatEvtsRatioClient = l1tStage2BmtfZeroSuppRatioClient.clone()
-l1tStage2BmtfZeroSuppFatEvtsRatioClient.monitorDir = cms.untracked.string(bmtfZSDqmDir+'/FatEvts')
-l1tStage2BmtfZeroSuppFatEvtsRatioClient.inputNum = cms.untracked.string(bmtfZSDqmDir+'/FatEvts/'+errHistNumStr)
-l1tStage2BmtfZeroSuppFatEvtsRatioClient.inputDen = cms.untracked.string(bmtfZSDqmDir+'/FatEvts/'+errHistDenStr)
-l1tStage2BmtfZeroSuppFatEvtsRatioClient.ratioTitle = cms.untracked.string('Summary of bad zero suppression rates')
-
+l1tStage2BmtfZeroSuppFatEvtsRatioClient = l1tStage2BmtfZeroSuppRatioClient.clone(
+monitorDir = bmtfZSDqmDir+'/FatEvts',
```

Comment:
please indent, here and everywhere else: would you do so, it will be also easier to spot where you forgot to modify the parameters inside the `clone(...)`

Reply:
Yes. I will remember this from now onwards. I am trying to indent wherever possible.